package-lock.json

```diff
@@ -991,7 +991,7 @@
     },
     "@sinonjs/formatio": {
       "version": "2.0.0",
-      "resolved": "http://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz",
+      "resolved": "https://registry.npmjs.org/@sinonjs/formatio/-/formatio-2.0.0.tgz",
```

Comment:
npm warns with low severity vulnerabilities. To fix with `npm audit fix`

Reply:
fixed